### PR TITLE
migrate code from googleapis/python-video-live-stream

### PR DESCRIPTION
**Pull request by parthea**
- docs(samples): add code samples (#21)
- chore(deps): update all dependencies (#33)
- chore(deps): update dependency google-cloud-video-live-stream to v0.1.2 (#34)
- ci: try to stop channel again before deleting it in the test (#36)
- docs(samples): add sample for creating a channel with a failover backup input (#37)
- chore(deps): update dependency google-cloud-video-live-stream to v0.1.3 (#39)
- chore: remove dependency google-api-python-client in samples (#40)
- chore(deps): update dependency grpcio to v1.46.0 (#41)
- chore(deps): update dependency grpcio to v1.46.1 (#44)
- chore(deps): update dependency grpcio to v1.46.3 (#47)
- chore(deps): update dependency google-cloud-video-live-stream to v0.1.4 (#48)
- docs: align channel config with best practices (#71)
- chore(deps): update all dependencies (#70)
- docs(samples): fix timeout for LROs; first call can take up to 10 minutes (#82)
- docs(samples): fix timeout for create input and start channel LROs; first call can take up to 15 minutes (#86)
- chore(deps): update all dependencies (#87)
- chore(deps): update all dependencies to v1.0.1 (#90)
- chore(deps): update dependency google-cloud-video-live-stream to v1.0.2 (#94)
- chore(deps): update dependency grpcio to v1.48.1 (#101)
- chore(deps): update dependency pytest to v7.1.3 (#106)
- chore(deps): update dependency grpcio to v1.49.0 (#111)
- chore(deps): update dependency grpcio to v1.49.1 (#112)
- chore(deps): update dependency google-cloud-video-live-stream to v1.0.3 (#115)
- chore(deps): update dependency google-cloud-video-live-stream to v1.0.4 (#118)
- chore(deps): update dependency grpcio to v1.50.0 (#119)
- chore(deps): update dependency pytest to v7.2.0 (#120)
- chore(deps): update dependency grpcio to v1.51.1 (#128)
- chore(deps): update dependency google-cloud-video-live-stream to v1.1.0 (#131)
- chore(deps): update dependency google-cloud-video-live-stream to v1.2.0 (#135)
- chore(deps): update dependency pytest to v7.2.1 (#136)
- chore(deps): update dependency google-cloud-video-live-stream to v1.2.1 (#139)
- chore(deps): update all dependencies (#146)
- chore(deps): update dependency pytest to v7.2.2 (#149)
- chore(deps): update all dependencies (#155)
- chore(deps): update dependency pytest to v7.3.0 (#156)
- chore(deps): update dependency pytest to v7.3.1 (#157)
- chore(deps): update dependency grpcio to v1.54.0 (#158)
- chore(deps): update dependency grpcio to v1.54.2 (#159)

## Description

Towards https://github.com/googleapis/google-cloud-python/issues/10966

Note: based on guidelines in dee-lang-mono-repos, use Rebase & Merge to preserve the commit history

### CODE REVIEW

**Comment by kweinmeister** [comment]
@parthea this is passing all tests; anything else needed?

File: video/live-stream/noxfile_config.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+# You can copy this file into your directory, then it will be imported from
+# the noxfile.py.
+
+# The source of truth:
+# https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/noxfile_config.py
+
+TEST_CONFIG_OVERRIDE = {
+    # You can opt out from the test for specific Python versions.
+    "ignored_versions": [],
```
Comment (leahecole) [inline video/live-stream/noxfile_config.py:25]:
```suggestion
    "ignored_versions": [2.7],
```

@dizcology re our chat

Comment (dizcology) [reply video/live-stream/noxfile_config.py:25]: Thanks - although I see no reason why this should not support Python 2.7, it should be fine to skip it.

**Comment by dizcology** [comment]
Still need to update CODEOWNERS and blunderbuss.  I will do this after https://github.com/GoogleCloudPlatform/python-docs-samples/pull/10605

File: video/live-stream/noxfile_config.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+# You can copy this file into your directory, then it will be imported from
+# the noxfile.py.
+
+# The source of truth:
+# https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/noxfile_config.py
+
+TEST_CONFIG_OVERRIDE = {
+    # You can opt out from the test for specific Python versions.
+    "ignored_versions": ["2.7"],
```
Comment (dandhlee) [inline video/live-stream/noxfile_config.py:25]: Is it fine to sync this file with the base noxfile_config for this repo? https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/noxfile_config.py

```suggestion
    "ignored_versions": ["2.7", "3.7", "3.9", "3.10"],
```

**Review by dandhlee — APPROVED** [review]
LGTM, one nit

File: video/live-stream/noxfile_config.py (hunk 2 above)
Comment (dizcology) [reply video/live-stream/noxfile_config.py]: That should be fine, thanks!

File: video/live-stream/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+To run the samples, you need to first follow the steps in
+[Before you begin](https://cloud.google.com/livestream/docs/how-to/before-you-begin).
+
+For more information on authentication, refer to the
+[Authentication Getting Started Guide](https://cloud.google.com/docs/authentication/getting-started).
+
+## Install Dependencies
+
+1. Clone python-video-live-stream and change directories to the sample directory
```
Comment (dandhlee) [inline video/live-stream/README.md:20]:
```suggestion
1. Clone python-docs-samples repository and change directories to the sample directory
```

File: video/live-stream/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+you want to use.
+
+        $ git clone https://github.com/googleapis/python-video-live-stream.git
```
Comment (dandhlee) [inline video/live-stream/README.md:23]:
```suggestion
        $ git clone https://github.com/GoogleCloudPlatform/python-docs-samples.git
```

**Review by dandhlee — CHANGES_REQUESTED** [review]
Forgot to check for this 🤪

**Comment by dizcology** [comment]
Thanks for catching this!  Doing a quick search for "googleapis/python-" turns up several other README files, by the way.

**Comment by dandhlee** [comment]
Perhaps we missed a few. I'll file an issue to help clean those up - thank you!

Admin merging this PR for sample migration.

**Review by dandhlee — APPROVED** [review]

**Pull request merged**